cmake/compiler/msvc/settings.cmake

```diff
@@ -84,3 +84,16 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Zm500")
 # 'function' : member function does not override any base class virtual member function
 # 'virtual_function' : no override available for virtual member function from base 'class'; function is hidden
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /we4263 /we4264")
+
+# Disable incremental linking in debug builds
```

Comment:
Perhaps worth adding to the comment that this is needed due to linking getting stuck on VS (which might be fixed in a later version).